app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -1,10 +1,10 @@
 module Hyrax
   class ThumbnailPathService
     class << self
-      # @param [Work, FileSet] object - to get the thumbnail for
+      # @param [#id] object - to get the thumbnail for
```

Comment:
Im wondering if this could be misleading. Its not an ID object right? and if its a method call you are referencing, you are calling #thumbnail_id. If Im understanding this right, this is a thumbnail object or a FileSet object right?

Reply:
the work of these changes is to ensure that the only interface for the object passed in is an `#id` instance method. in YARD convention, that's what this means. in the case of `#thumbnail_id`, this PR changes the service to give the default thumbnail path if it's not implemented.

in practice, what we plan to pass in is any ActiveFedora::Base or any Valkyrie::Resource object. but again, the goal of this change is to codify the contract of this service as requiring `#id` only.